homeassistant/exceptions.py

```diff
@@ -13,6 +13,9 @@
 class HomeAssistantError(Exception):
     """General Home Assistant exception occurred."""
 
+    _message: str | None = None
+    _translate: bool = False
```

Comment:
Other sub classes can set this flag to ensure translations to be fetched, even if there are `*args` set.

Reply:
It is needed as `self.args` is overridden by the super class and is not suitable not be used to check if translations are wanted.